camligorkem/assignment3/Dictionary.java

```diff
@@ -0,0 +1,77 @@
+import java.util.HashMap;
+import java.util.Map;
+
+// Trie Node class to create Dictionary based on prefixes
+class TrieNode{
+    Map<Integer, TrieNode> letters;
```

Comment:
Now you can as well map character not integers, it will make your code much more readable.